core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -161,13 +161,6 @@ export class ContributionsAndReview implements OnInit, OnDestroy {
   };
 
   opportunities: ExplorationOpportunitySummary[] = [];
-
-  /**
-   * The feature flag state to gate the contributor_dashboard_accomplishments.
-   * @type {boolean} - contributor_dashboard_accomplishments - A boolean value.
-   * This determines whether the contributor_dashboard_accomplishments feature
-   * is enabled.
-   */
   accomplishmentsTabIsEnabled: boolean = false;
```

Comment:
The variable should be removed, as it is no longer being referenced.

Reply:
noted ser, will try to do it at my earliest